xml/art_sle_ha_install_quick.xml

```diff
@@ -529,22 +547,22 @@ info: Received command test from &node1; on disk <replaceable>SBD</replaceable><
     <para>
      Start the bootstrap script by executing:
     </para>
-<screen>&prompt.root;<command>ha-cluster-init</command></screen>
-    <para>
-     If NTP has not been configured to start at boot time, a message
-     appears. The script also checks for a hardware watchdog device (which
-     is important in case you want to configure SBD) and warns you if none
-     is present.
-    </para>
-    <remark>toms, 2016-07-30: (kai) AFAIk ha_init offers NTP configuration/start
-      if it does not run before.
-      If this is the case, then the sentence about NTP can be dropped. (As the
-      screen will tell/help in case it isn't - save some electrons/characters).</remark>
-    <para>
-     If you decide to continue anyway, the script will automatically
-     generate keys for SSH access and for the &csync; synchronization
-     tool and start the services needed for both.
-    </para>
+    <screen>&prompt.root;<command>ha-cluster-init</command></screen>
+    <para>After confirming with <literal>Y</literal>, the script does:</para>
```

Comment:
You only need to confirm with yes if NTP is not configured. In case it is already configured, the scripts just starts with its tasks. Therefore better rephrase to: 
    <para>The scripts checks for NTP configuration and a hardware watchdog device. It generates the 
    public and private SSH keys used for SSH access and &csync; synchronization and starts the respective services.</para>